lib/sycamore/sycamore/data/document.py

```diff
@@ -1,14 +1,13 @@
 from collections import UserDict
 import json
-from enum import Enum
 from typing import Any, Optional
 import uuid
 
 from sycamore.data import BoundingBox, Element
 from sycamore.data.element import create_element
 
 
-class DocumentSource(Enum):
+class DocumentSource:
```

Comment:
Can't you use this?

https://docs.pydantic.dev/2.0/usage/types/enums/

Reply:
Seems like it wants me to convert Document to a BaseModel too then?

Reply:
I'm going to skip this for now until we have an actual need for an enum